src/gobstuf/rest/brp/base_view.py

```diff
@@ -0,0 +1,152 @@
+from flask.views import MethodView
+from flask import request, abort, Response
+from flask_hal.document import Document
+from requests.exceptions import HTTPError
+from abc import abstractmethod
+
+from gobstuf.certrequest import cert_post
+from gobstuf.stuf.brp.base_request import StufRequest
+from gobstuf.stuf.exception import NoStufAnswerException
+from gobstuf.stuf.brp.error_response import StufErrorResponse
+from gobstuf.config import ROUTE_SCHEME, ROUTE_NETLOC, ROUTE_PATH
+
+
+MKS_USER_HEADER = 'MKS_GEBRUIKER'
+MKS_APPLICATION_HEADER = 'MKS_APPLICATIE'
+
+
+def headers_required_decorator(headers):
+    def headers_required(f):
+        def decorator(*args, **kwargs):
+            if not all([request.headers.get(h) for h in headers]):
+                return abort(Response(response='Missing required MKS headers', status=400))
+
+            return f(*args, **kwargs)
+        return decorator
+    return headers_required
+
+
+class StufRestView(MethodView):
+    """StufRestView.
+
+    Maps a GET request with URL parameters defined in kwargs to an MKS StUF request.
+
+    Should be extended with a request_template and response_template.
+    """
+    decorators = [headers_required_decorator([MKS_USER_HEADER, MKS_APPLICATION_HEADER])]
+
```

Comment:
I suggest to add some comment that explain the use of decorators in a Flask MethodView. Now it looks like its unused code if you quickly pass over it.

Reply:
Done